recipes/pysam/meta.yaml

```diff
@@ -29,17 +34,19 @@ requirements:
     - openssl  # [not osx]
   run:
     - python
-    - zlib
-    - xz
-    - bzip2
-    - libdeflate
-    - libcurl
-    - openssl  # [not osx]
+
+test:
+  imports:
+    - pysam
 
 about:
-  home: https://github.com/pysam-developers/pysam
-  license: MIT
+  home: https://pypi.org/project/pysam/
```

Comment:
What is the reason for updating `home` away from the place where the upstream pysam maintainers work?